### PR TITLE
Adding stores to organization settings

### PR DESCRIPTION
**Pull request by giancorzo**
https://developertown.visualstudio.com/SIL%20International%20Scriptoria/_workitems/edit/29293

- [x] Feature
- [x] Tests

stores

### CODE REVIEW

**Comment by NullVoxPopuli** [comment]
Should this share styles with this view? https://app.zeplin.io/project/5b3b85b95b0fc79a4b3c40c9/screen/5b3b8d86a84659a2a2edaf0f
(or rather, should that view share styles with this view?)

File: source/SIL.AppBuilder.Portal.Frontend/src/ui/routes/organizations/settings/index.tsx [hunk 1]
```diff
@@ -86,6 +87,19 @@ class SettingsRoute extends React.Component<IProps> {
     }
   }
 
+  updateOrganizationStore = async (st) => {
+
+    const { t, updateStore } = this.props;
+
+    try {
+
```
Comment (NullVoxPopuli) [inline source/SIL.AppBuilder.Portal.Frontend/src/ui/routes/organizations/settings/index.tsx]: I'm all for vertical space and everything, but this looks silly

File: source/SIL.AppBuilder.Portal.Frontend/src/ui/routes/organizations/settings/stores/__tests__/acceptance-test.tsx [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+          beforeEach(function () {
+            this.mockDelete(204, 'organization-stores/1');
+          });
+
+          beforeEach(async function () {
+            const storeList = page.stores();
+            await storeList[0].click();
+          });
+
+          it('first store is un selected', () => {
```
Comment (NullVoxPopuli) [inline source/SIL.AppBuilder.Portal.Frontend/src/ui/routes/organizations/settings/stores/__tests__/acceptance-test.tsx]: unselected is one word :)
Also, this is checking the value of two stores?

File: source/SIL.AppBuilder.Portal.Frontend/src/ui/routes/organizations/settings/stores/index.tsx [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+type IOwnProps =
+  & IProps
+  & i18nProps;
+
+const mapRecordsToProps = (passedProps) => {
+  const { organization } = passedProps;
+  const { type, id } = organization;
+
+  return {
+    organizationStores: q => q.findRelatedRecords({ type, id }, 'organizationStores')
```
Comment (NullVoxPopuli) [inline source/SIL.AppBuilder.Portal.Frontend/src/ui/routes/organizations/settings/stores/index.tsx]: any reason to not just pass the organization here?

Comment (giancorzo) [reply source/SIL.AppBuilder.Portal.Frontend/src/ui/routes/organizations/settings/stores/index.tsx]: changed

**Comment by giancorzo** [comment]
this how I'm getting the organizationStores

const mapRecordsToProps = (passedProps) => {

  const { organization } = passedProps;

  return {
    organizationProductDefinitions: q =>
      q.findRelatedRecords(organization, 'organizationProductDefinitions'),
    organizationStores: q =>
      q.findRelatedRecords(organization, 'organizationStores')
  };
};

**Pull request merged**